pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -1134,6 +1134,10 @@ func (dm *DockerManager) checkVersionCompatibility() error {
 }
 
 func (dm *DockerManager) fmtDockerOpts(opts []dockerOpt) ([]string, error) {
+	if opts == nil {
```

Comment:
Could you add a comment explaining why this is necessary? That'd help preventing regression.